eclair-core/src/test/scala/fr/acinq/eclair/channel/states/h/ClosingStateSpec.scala

```diff
@@ -797,6 +856,27 @@ class ClosingStateSpec extends TestkitBaseClass with StateTestsHelperMethods {
     awaitCond(alice.stateData.asInstanceOf[DATA_CLOSING].copy(revokedCommitPublished = Nil) == initialState)
   }
 
+
+  test("recv BITCOIN_FUNDING_SPENT (one revoked tx, option_static_remotekey)", Tag("static_remotekey")) { f =>
```

Comment:
Same as others, I think it would be great to factorize these tests so that we run it in both version (no static_remotekey / static_remotekey) without duplicating too much code.

Reply:
Done in 4abf88d